### ISSUE TITLE
Basic/WithFamilyArgs.DatagramSend/1 failed

### ISSUE THREAD

**Issue by nibanks** [microsoft/msquic#3561]
### Describe the bug

https://github.com/microsoft/msquic/actions/runs/4679806246/jobs/8290342265
```
[----------] 1 test from Basic/WithFamilyArgs
[ RUN      ] Basic/WithFamilyArgs.DatagramSend/1
D:\a\msquic\msquic\src\test\lib\DatagramTest.cpp(265): error: Client.GetDatagramsLost() >= 1 not true

[  FAILED  ] Basic/WithFamilyArgs.DatagramSend/1, where GetParam() = v6 (783 ms)
[----------] 1 test from Basic/WithFamilyArgs (784 ms total)
```

### Affected OS

- [X] Windows
- [ ] Linux
- [ ] macOS
- [ ] Other (specify below)

### Additional OS information

Was with XDP and QTIP. Not sure if that matters.

### MsQuic version

main

### Steps taken to reproduce bug

Run tests

### Expected behavior

Tests all pass

### Actual outcome

Occasional test failure

### Additional details

[quic.log](https://github.com/microsoft/msquic/files/11222466/quic.log)

**Comment by nibanks** [comment]
It seems to be an interaction with PMTUD delaying the loss notification. We need to disable PMTUD on these tests.

**Comment by nibanks** [comment]
Fixed by #3563.
